tests/integration/modules/test_pip.py

```diff
@@ -40,6 +40,48 @@ def cleanup_environ(environ):
 
         self.addCleanup(cleanup_environ, os.environ.copy())
 
+    def test_issue_2087_missing_pip(self):
+        # Run a pip depending functions with a non existent pip binary
+        for func in ('pip.freeze', 'pip.list'):
+            ret = self.run_function(func, bin_env=os.path.join(RUNTIME_VARS.TMP, 'no_pip'))
+            self.assertNotIsInstance(ret, list)
+            self.assertNotIsInstance(ret, dict)
+            self.assertIn('command required for \'{0}\' not found: '.format(func), ret.lower())
+            self.assertIn('could not find a pip binary', ret)
+
+    @skipIf(salt.utils.path.which('deactivate'), 'Must not be in a virtual environment for this test')
+    @destructiveTest
+    def test_system_pip3(self):
```

Comment:
Why does it have to be the system's pip3?
We'll never run these under nox, it's always a virrualenv...

Reply:
@soundtech Should the test be removed then if it will always be a virtualenv?  If it's always going to be skipped because it will always run in a virtualenv it isn't valuable

Reply:
Well, Nox runs tests in a virtualenv, but salt usually works against the system python, so, my question is rather, does it matter if its the system pip or the virtualenv pip.
I'll have a better look at the test tomorrow since I'm writing from the phone and the tests could actually be really explicit as to why.

Reply:
@s0undt3ch  It seems to me that this test is using the environment's pip to test that salt actually installed things correctly through pip.  I don't think the test makes sense to leave in there.

Reply:
Do we have another test that covers installing a python package?

Reply:
yes, within a virtual environment, I don't actually see the point of testing this with the system pip vs a virtualenv pip